CHANGELOG.md

```diff
@@ -4,6 +4,104 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.0.0-beta.0
+
+This is the first beta release of Sentry JavaScript SDK v8. With this release, there are no more planned breaking
+changes for the v8 cycle.
+
+Read the [in-depth migration guide](./MIGRATION.md) to find out how to address any breaking changes in your code. All
```

Comment:
could we add notes about version changes and package removals here? https://github.com/getsentry/sentry-javascript/pull/11410/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR7

Reply:
will add the stuff you wrote there here too! 👍

Reply:
we added this now 👍